### PR TITLE
Add build phase to script - @samtgarson

### PR DESCRIPTION
**Pull request by califa**
This PR adds a build phase to the script.

---

There are a few things that are either wrong or could be improved: 

1. On lines 164 and 202, `TEMP_HTML_PATH` should be camelCased like the other constants in the file 
2. On line 168, the accidental assignment `=` should actually be a comparison `===`
3. Also on line 168, the opening conditional bracket `{` is missing 

Suggest the appropriate changes.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -155,6 +155,59 @@ async function main () {
   }
 }
 
+/*
+ * ==============================================================
+ *                         BUILD
+ * ==============================================================
+ */
+
+const TEMP_HTML_PATH = path.join(tempDir, inputFilenameNoExt + '_temp.htm')
```
Comment (califa) [inline script.js]:
```diff
-const TEMP_HTML_PATH = path.join(tempDir, inputFilenameNoExt + '_temp.htm')
+const TEcgcgcPATH = path.join(tempDir, inputFilenameNoExt + '_temp.htm')
```

Comment (samtgarson) [inline script.js:164]: This constant should be `camelCased` not `UNDER_SCORED`, as on line 202

```diff
-const TEMP_HTML_PATH = path.join(tempDir, inputFilenameNoExt + '_temp.htm')
+const tempHtmlPath = path.join(tempDir, inputFilenameNoExt + '_temp.htm')
```

File: script.js [hunk 2]
```diff
@@ -155,6 +155,59 @@ async function main () {
...
+
+async function build (filepath) {
+  var shortFileName = filepath.replace(inputDir, '')
+  if ((path.basename(filepath) = 'config.yml') || (filepath.endsWith('.plugin.js')))
```
Comment (samtgarson) [inline script.js]:
```diff
-  if ((path.basename(filepath) = 'config.yml') || (filepath.endsWith('.plugin.js')))
+  if ((path.basename(filepath) === 'config.yml') || (filepath.endsWith('.plugin.js'))) {
```

triple `===` and missing opening `{` 👍

**Review by samtgarson — CHANGES_REQUESTED** [review]
Looks good in general but couple of small comments

File: script.js (hunk 1 above)
Comment (samtgarson) [reply script.js:164]: constants should be uppercased...

**Comment by samtgarson** [comment]
😞

**Pull request closed without merging**